addons/website/views/snippets/snippets.xml

```diff
@@ -410,6 +410,37 @@
          data-selector='.accordion > .card'
          data-drop-in='.accordion:has(> .card)'/>
 
+    <div data-js="Accordion"
+         data-selector='.s_faq_collapse'>
+        <we-select string="Card Style">
+            <we-button title="Default"
+             data-select-class="">
+                Default
+            </we-button>
+            <we-button title="Light" 
+             data-select-class="s_faq_collapse_light">
+                Light
+            </we-button>
+            <we-button title="Boxed"
+             data-select-class="s_faq_collapse_boxed">
+                Boxed
+            </we-button>
+            <we-button title="Big"
+             data-select-class="s_faq_collapse_big">
+                Big
+            </we-button>
+        </we-select>
+        <we-button-group string="Collapse Icon" data-no-preview="true">
+            <we-button data-icon-pos="left"
+             data-img="/website/static/src/img/snippets_options/pos_left.svg"
+             title="Left"/>
+            <we-button data-icon-pos="right"
+             data-img="/website/static/src/img/snippets_options/pos_right.svg"
+             title="Right"/>
+        </we-button-group>
+    </div>
```

Comment:
For options specific to a specific snippet the options should be put in an options.xml file in the related snippet folder (check other snippets for examples).

Reply:
Do you mean inside views/snippets/$snippet_name$.xml ? This is the only place where I see compartmentalized snippet options for a specific snippets; I can't find any folders containing an options.xml file. A good candidate would have been static/src/snippets/$snippet_name$/options.xml, but I only see some instances of an options.js file, as well as some versioned 00X.xml files defining some templates.

Reply:
Yes, I was confused with options.js but it should indeed be in the XML file where the snippet itself is defined.